src/components/Footer.tsx

```diff
@@ -1,21 +1,29 @@
+import { createSharedPathnamesNavigation } from 'next-intl/navigation';
+import { locales } from '@/locales';
 import { useTranslations } from 'next-intl';
 
+const { Link } = createSharedPathnamesNavigation({
+  locales: [...locales.values()],
+});
+
 export default function Footer({ className }: { className?: string }) {
   const t = useTranslations('Footer');
   return (
-    <footer className={`footer bg-black py-2 d-flex flex-wrap ${className ?? ''}`}>
+    <footer className={`footer bg-black py-3 d-flex flex-wrap ${className ?? ''}`}>
       <div className="container-sm d-flex flex-wrap justify-content-around">
         <a
-          href="#"
+          href="/imprint"
           className="lead link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover"
         >
-          {t('imprint')}
+          {t('imprint')} & <br /> {t('dataProtection')}
```

Comment:
Is there a reason for having the `<br>`? We could just keep it on a single line, I think. Also, we could consider reducing the text size. In my opinion, it should be smaller than the text on the main page. Maybe add a class `fs-5` or `fs-6`.